woocommerce/woocommerce-android
# Feature/localize store profiler

### Description
Currently, we don't have an API available to load store profiler option from backend. So, we are hardcoding this options inside [StoreProfilerRepository](https://github.com/woocommerce/woocommerce-android/blob/636261e58d3aafbc0209719d038dedd023e83acb/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/profiler/StoreProfilerRepository.kt). 

In this PR I'm adding support to have all these hardcoded option localized. 

### Testing instructions
Currently, none of the new strings I've added are available in other languages, as they require translations from GlotPress. But testing that the store profiler options are correctly displayed in English should suffice. To achieve that, trigger store creation flow and navigate to store profiler steps.

**Note**: To trigger the store creation flow you'll need to compile vanillaDebug flavour, and if the Google account you've set on your device is not US located you'll also need to disable the feature flag IAP_FOR_STORE_CREATION for debug builds.
